libpurecool/dyson.py

```diff
@@ -52,6 +52,23 @@ def login(self):
         urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
         _LOGGER.debug("Disabling insecure request warnings since "
                       "dyson are using a self signed certificate.")
+        # Must first check account status
+        accountstatus = requests.get(
+            "https://{0}/v1/userregistration/userstatus".format(self._dyson_api_url),
+            params={"country": self._country, "email": self._email},
+            headers=self._headers,
+            verify=False,
+        )
```

Comment:
```
accountstatus = requests.get(
             "https://{0}/v1/userregistration/userstatus".format(
                self._dyson_api_url
             ),
             params={"country": self._country, "email": self._email},
             headers=self._headers,
             verify=False,
         )
```
maybe change this line so it's shorten and will pass the checks

Reply:
I'm not really sure what to do about that - It's cause of the indents in the line(s) iirc and if i just un-indent them it'll look really weird.

Reply:
Follow the patterns of lines 78-79?

Reply:
Yes, just break apart

```python
            "https://{0}/v1/userregistration/userstatus".format(self._dyson_api_url),
```
to be
```python
            "https://{0}/v1/userregistration/userstatus".format(
                self._dyson_api_url
            ),
```

Reply:
Ah good point - done in 3965d81